src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -1,32 +1,40 @@
 package com.kodilla.ecommercee.domain;
 
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
 
+
+@Entity(name = "ORDERS")
 @Getter
-@AllArgsConstructor
+@Setter
 @NoArgsConstructor
-@Entity(name = "ORDERS")
 public class Order {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     @Column(name = "ID", unique = true)
     private Long id;
 
-//    @OneToOne(
-//            cascade = CascadeType.PERSIST,
-//            fetch = FetchType.LAZY
-//    )
-//    @JoinColumn(name = "CART_ID")
-//    private Cart cart;
+    @OneToMany(
+            targetEntity = Cart.class,
+            mappedBy = "order",
+            cascade = CascadeType.ALL,
+            fetch = FetchType.EAGER
+    )
```

Comment:
Pytanie czy w tym przypadku, warto dawać CascadeType.ALL, pytanie czy nie warto zrezygnować z Remove. Pytanie czy warto przy usuwaniu koszyka usuwać zamówienie i na odwrót. Ciężko mi powiedzieć, co wy na to ?